docs/content/v0.20/README.md

```diff
@@ -74,12 +75,12 @@ helm --kube-context imbs1 upgrade --install status-addon -n open-cluster-managem
 
 see [here](https://github.ibm.com/dettori/status-addon) for more details on the add-on.
 
-4. Create a Workload Description Space `wds1` directly in KubeFlex. Similarly to before, `-p kslight` 
+4. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar` 
```

Comment:
Would it make sense to document how to run a local build of kubestellar rather than use the -p kubestellar param?

Reply:
yes, definitely should be documented, but perhaps it should be on a "developer guide" doc ? This doc (or what we will derive from it) should be focused on end users that want to test a release without having to clone the project, install go and build tools and make builds to run the tests.